README.md

```diff
@@ -1 +1,20 @@
-# hw01
\ No newline at end of file
+# About Cheyenne
+
+![Image of me](https://github.com/cheyennewh/hw01/blob/master/selfie.png)
```

Comment:
Nice inclusion! You can also use simply `![Image of me](selfie.jpg)` to embed the image since the README is in the same folder